src/EditorFeatures/CSharpTest/CodeActions/GenerateFromMembers/GenerateConstructor/GenerateConstructorTests.cs

```diff
@@ -93,7 +93,7 @@ protected override object CreateCodeRefactoringProvider(Workspace workspace)
             await TestAsync(
 @"class Z { [|public int A { get ; private set ; } public string B { get ; private set ; }|] } ",
 @"class Z { public Z ( int a , string b ) { this . A = a ; this . B = b ; } public int A { get ; private set ; } public string B { get ; private set ; } } ",
-index: 0, options: new Dictionary<OptionKey, object> { { new OptionKey(SimplificationOptions.QualifyMemberAccessWithThisOrMe, "C#"), true } });
+index: 0, options: new Dictionary<OptionKey, object> { { new OptionKey(SimplificationOptions.QualifyPropertyAccess, LanguageNames.CSharp), true } });
```

Comment:
Please add some sort o helper that makes this options object for you.  i.e. ```Options(QuaifyPropertyAccess, true)```  And then use that all over the place.